### PR TITLE
Update Server.gd

### PR DESCRIPTION
**Pull request by INdIEDeV016**
This line is unnecessary. See `member_button.gd`, It is already there in it

### CODE REVIEW

**Comment by INdIEDeV016** [comment]
Also, no need to PR  your commit now. But please, don't commit directly from next time!

File: Scenes/Server/Server.gd [hunk 1]
```diff
@@ -29,6 +29,5 @@ func _ready() -> void:
 	for member in members:
 		var member_button = member_button_scene.instance()
 		member_button.name = member["user"]["username"]
-		member_button.text = member["user"]["username"]
```
Comment (INdIEDeV016) [inline Scenes/Server/Server.gd:32]: This is the line I am talking about! No need, see `member_button.gd`, it is already there!

**Comment by sairam4123** [comment]
I thought that was a bug

**Comment by sairam4123** [comment]
Also I would not recommend using names as button name, instead use IDs.

**Pull request merged**